mobile/native/src/orderbook.rs

```diff
@@ -176,11 +178,13 @@ pub fn subscribe(
                 tracing::warn!("Cannot update orderbook status: {e:#}");
             };
 
-            tokio::time::sleep(WS_RECONNECT_TIMEOUT).await;
+            let retry_interval = Duration::from_millis(WS_RECONNECT_TIMEOUT) * round;
```

Comment:
🙃 I suppose we are going for some sort of backoff here. We could use [this](https://docs.rs/backoff/latest/backoff/backoff/trait.Backoff.html) library that we've used in the past which is a bit smarter and can use [exponential backoff](https://docs.rs/backoff/latest/backoff/exponential/struct.ExponentialBackoff.html).

Reply:
I had a quick look at the library and it feels like I would need to make far more changes to make it work with that library than I would like to invest at the moment.